resources/ansible.yaml

```diff
@@ -286,3 +286,18 @@ resources:
         - redhat-cop/cloud.terraform_ops:
             connection: github.com-git
             zuul/include: []
+        - ansible-collections/ansible_hub:
+            zuul/include: []
+        - ansible-collections/ansible/event-driven-ansible:
```

Comment:
I’m not totally sure, but event-driven-ansible might not be necessary because @ssbarnea implemented the publishing to galaxy and ah in the GitHub CI. Can you please clarify @ssbarnea?

Reply:
We can remove that needed, I understand there's a time sensitive aspect to enabling the ability to release these